### PR TITLE
downloader.py updates

### PR DESCRIPTION
**Pull request by btv**
* minor syntax tweaks
* modified `is_supported` to use a tuple and return the result directly.
* changed conditional.

### CODE REVIEW

File: src/opnsense/scripts/suricata/lib/downloader.py [hunk 1]
```diff
@@ -204,16 +205,13 @@ def download(self, url, url_filename, filename, auth=None, headers=None, version
                         target_filename, fetch_result['filename']
                 ))
                 return None
-            if not fetch_result['cached']:
+            if 'cached' not in fetch_result:
```
Comment (AdSchellevis) [inline src/opnsense/scripts/suricata/lib/downloader.py:208]: `cached` is a boolean, in which case the above statement would return true when false.

Comment (btv) [reply src/opnsense/scripts/suricata/lib/downloader.py]: Sorry for the delay on this and thanks you for the review. I've updated the code based on your comment.

Comment (AdSchellevis) [reply src/opnsense/scripts/suricata/lib/downloader.py]: @btv no problem, we're all busy :)

**Comment by AdSchellevis** [comment]
@btv thanks!

**Pull request merged**